lib/inputs/rpm/static.ts

```diff
@@ -38,3 +45,42 @@ export async function getRpmDbFileContent(
     return "";
   }
 }
+
+export async function getRpmSqliteDbFileContent(
+  extractedLayers: ExtractedLayers,
+): Promise<PackageInfo[]> {
+  const rpmDb = getContentAsBuffer(
+    extractedLayers,
+    getRpmSqliteDbFileContentAction,
+  );
+  if (!rpmDb) {
+    return [];
+  }
+
+  try {
+    const fileObj = tmp.fileSync();
```

Comment:
No it isn't. We need it here specifically because we don't analyze it as plain text as we need to open the sqlite db - for that we need a file.
Will change the name :)